support-frontend/assets/pages/supporter-plus-landing/components/summaryTsAndCs.tsx

```diff
@@ -63,9 +64,13 @@ export function SummaryTsAndCs({
 		productKey,
 		ratePlanKey,
 	);
+	const isPaperSundayOrPlus =
+		isSundayOnlyNewsletterSubscription ||
+		isPaperPlusSub(productKey, ratePlanKey);
```

Comment:
Could this be simplified to just `isPaperSub` or similar now? This should be true for all paper products we sell through the site I think?

Reply:
I think the `ActiveRatePlanKey` still contains non-plus products, thought best to leave until thats cleaned up in a seperate PR and plus'es can be removed throughout then?